src/django/api/serializers.py

```diff
@@ -831,13 +831,7 @@ def get_contributor_fields(self, facility):
         if not embed == '1' or contributor_id is None:
             return []
 
-        # If the contributor has not created any overriding embed config these
-        # transparency pledge fields will always be visible.
-        fields = [
-            EmbedField(column_name=column_name, display_name=display_name)
-            for (column_name, display_name)
-            in NonstandardField.DEFAULT_FIELDS.items()]
-
+        fields = []
```

Comment:
We remove handling for extended fields from the get_contributor_fields serializer method because the values are now retrieved as by the get_extended_fields serializer method.